src/components/collections/CollectionHomeHeader.jsx

```diff
@@ -32,33 +32,43 @@ import ProcessingChip from '../common/ProcessingChip';
 import ConceptContainerDelete from '../common/ConceptContainerDelete';
 import CollapsibleDivider from '../common/CollapsibleDivider';
 
-const HIDDEN_ATTRIBUTES = {
-  canonical_url: 'url',
-  publisher: 'text',
-  purpose: 'text',
-  copyright: 'text',
-  preferred_source: 'text',
-  custom_resources_linked_source: 'text',
-  revision_date: 'date',
-  identifier: 'json',
-  contact: 'json',
-  jurisdiction: 'json',
-  meta: 'json',
-  immutable: 'boolean',
-  locked_date: 'date',
-  experimental: 'boolean'
-}
+const DEFAULT_VISIBLE_ATTRIBUTES = [
```

Comment:
shouldn't this come from DEFAULT_CONFIG? why redefine here?

Reply:
These are the attributes if not defined in the visibleAttributes should go to invisible by default.